kedro/framework/cli/micropkg.py

```diff
@@ -0,0 +1,916 @@
+            )
+        # Extract package name, based on the naming convention for wheel files
+        # https://www.python.org/dev/peps/pep-0427/#file-name-convention
+        package_name = dist_info_file[0].stem.split("-")[0]
+        package_metadata = dist_info_file[0] / "METADATA"
+
+        req_pattern = r"Requires-Dist: (.*?)\n"
+        package_reqs = re.findall(req_pattern, package_metadata.read_text())
+        if package_reqs:
+            requirements_in = _get_requirements_in(
+                metadata.source_dir, create_empty=True
+            )
+            _append_package_reqs(requirements_in, package_reqs, package_name)
+
+        _clean_pycache(temp_dir_path)
+        _install_files(metadata, package_name, temp_dir_path, env, alias)
+
+
+def _pull_packages_from_manifest(metadata: ProjectMetadata) -> None:
+    # pylint: disable=import-outside-toplevel
+    import anyconfig  # for performance reasons
+
+    config_dict = anyconfig.load(metadata.config_file)
+    config_dict = config_dict["tool"]["kedro"]
+    build_specs = config_dict.get("micropkg", {}).get("pull")
+
+    if not build_specs:
+        click.secho(
+            "Nothing to pull. Please update the `pyproject.toml` package manifest section.",
+            fg="yellow",
+        )
+        return
+
+    for package_path, specs in build_specs.items():
+        if "alias" in specs:
+            _assert_pkg_name_ok(specs["alias"])
+        _pull_package(package_path, metadata, **specs)
+        click.secho(f"Pulled and unpacked `{package_path}`!")
+
+    click.secho("Pipelines pulled and unpacked!", fg="green")
+
+
+def _package_pipelines_from_manifest(metadata: ProjectMetadata) -> None:
+    # pylint: disable=import-outside-toplevel
+    import anyconfig  # for performance reasons
+
+    config_dict = anyconfig.load(metadata.config_file)
+    config_dict = config_dict["tool"]["kedro"]
+    build_specs = config_dict.get("micropkg", {}).get("package")
+
+    if not build_specs:
+        click.secho(
+            "Nothing to package. Please update the `pyproject.toml` package manifest section.",
+            fg="yellow",
+        )
+        return
+
+    for pipeline_name, specs in build_specs.items():
+        if "alias" in specs:
+            _assert_pkg_name_ok(specs["alias"])
+        _package_pipeline(pipeline_name, metadata, **specs)
+        click.secho(f"Packaged `{pipeline_name}` pipeline!")
+
+    click.secho("Pipelines packaged!", fg="green")
+
+
+@micropkg.command("package")
+@env_option(
+    help="Environment where the pipeline configuration lives. Defaults to `base`."
+)
+@click.option(
+    "--alias",
+    type=str,
+    default="",
+    callback=_check_pipeline_name,
+    help="Alternative name to package under.",
+)
+@click.option(
+    "-d",
+    "--destination",
+    type=click.Path(resolve_path=True, file_okay=False),
+    help="Location where to create the wheel file. Defaults to `src/dist`.",
```

Comment:
Since this command is new, we can directly make the default to be `dist/` rather than `src/dist/` (no need to keep it as it is for backwards compatibility).

Reply:
I think this still needs to be updated right?